generators/app/index.js

```diff
@@ -589,12 +698,68 @@ module.exports = class extends Generator {
             case 'ext-localization':
                 localization.writingLocalizationExtension(this);
                 break;
+            case 'ext-notebook':
+                this._writingNotebook();
+                break;
             default:
                 //unknown project type
                 break;
         }
     }
 
+    _writingNotebook() {
+
+        let context = this.extensionConfig;
+
+        if (context.addNotebooks) {
+            for (let i = 0; i < context.notebookPaths.length; i++) {
+                this.fs.copy(context.notebookPaths[i], context.name + '/' + context.notebookNames[i]);
+            }
+        } else {
+            if (context.notebookType === 'notebook-python') {
+                this.fs.copy(this.sourceRoot() + '/optional/pySample.ipynb', context.name + '/pySample.ipynb');
```

Comment:
use path join not add strings to create path. the code should work in Windows, Mac and Linux